### PR TITLE
Harshad_TreeProgram_25min

### PR DESCRIPTION
**Pull request by HarshadBJun20**
-> TreeProgram done

### CODE REVIEW

File: src/harshadBJun20/CodingExams/TreeProgram.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package harshadBJun20.CodingExams;
+
+/*[3,5,5,11,9,10,13] --> 4*/
+public class TreeProgram {
+	int max,count;
```
Comment (KrishnaKTechnocredits) [inline src/harshadBJun20/CodingExams/TreeProgram.java:5]: count = 1

File: src/harshadBJun20/CodingExams/TreeProgram.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+	void countNumOfTreesVisible(int[] arr) {
+		max=arr[0];
+		for(int index=0;index<arr.length;index++) {
+			if(arr[index]>=max) {
```
Comment (KrishnaKTechnocredits) [inline src/harshadBJun20/CodingExams/TreeProgram.java]: if(arr[index]>max)

File: src/harshadBJun20/CodingExams/TreeProgram.java [hunk 3]
```diff
@@ -0,0 +1,22 @@
+package harshadBJun20.CodingExams;
+
+/*[3,5,5,11,9,10,13] --> 4*/
+public class TreeProgram {
+	int max,count;
+	void countNumOfTreesVisible(int[] arr) {
+		max=arr[0];
+		for(int index=0;index<arr.length;index++) {
```
Comment (KrishnaKTechnocredits) [inline src/harshadBJun20/CodingExams/TreeProgram.java]: index = 1

File: src/harshadBJun20/CodingExams/TreeProgram.java (hunk 1 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/CodingExams/TreeProgram.java:5]: Changes done

File: src/harshadBJun20/CodingExams/TreeProgram.java (hunk 2 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/CodingExams/TreeProgram.java]: Changes done.

File: src/harshadBJun20/CodingExams/TreeProgram.java (hunk 3 above)
Comment (HarshadBJun20) [reply src/harshadBJun20/CodingExams/TreeProgram.java]: Changes done

**Pull request merged**